extensions/azurecore/src/extension.ts

```diff
@@ -135,6 +135,12 @@ export async function activate(context: vscode.ExtensionContext): Promise<azurec
 			ignoreErrors: boolean,
 			query: string): Promise<azurecore.ResourceQueryResult<T>> {
 			return azureResourceUtils.runResourceQuery(account, subscriptions, ignoreErrors, query);
+		},
+		runGetRequest(account: azdata.Account,
```

Comment:
make the name more specific to what we are actually fetching by making the HTTP Request here.